exec-analysis/src/main/java/com/capitalone/dashboard/exec/config/LdapHandler.java

```diff
@@ -0,0 +1,67 @@
+package com.capitalone.dashboard.exec.config;
+
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Scope;
+import org.springframework.ldap.core.LdapTemplate;
+import org.springframework.ldap.core.support.LdapContextSource;
+import org.springframework.ldap.filter.EqualsFilter;
+import org.springframework.ldap.filter.Filter;
+import org.springframework.stereotype.Component;
+
+@Component
+@Scope(value = "singleton")
```

Comment:
default scope is singleton, hence this is redundant

Reply:
Removed scope to avoid redundant.